apps/server/src/modules/user-login-migration/service/user-login-migration.service.spec.ts

```diff
@@ -587,7 +589,7 @@ describe('UserLoginMigrationService', () => {
 
 			it('should return UserLoginMigration with start date and target system', async () => {
 				const { schoolId, targetSystemId } = setup();
-				const expected: UserLoginMigrationDO = new UserLoginMigrationDO({
+				const expected: UserLoginMigrationDO = userLoginMigrationDOFactory.buildWithId({
 					id: new ObjectId().toHexString(),
```

Comment:
Good point to use the factory instead of the constructor. But do you need to set the id if you use buildWithId?